lib/spack/spack/test/cmd/module.py

```diff
@@ -83,6 +83,25 @@ def test_remove_and_add_tcl(database, parser):
         assert os.path.exists(item)
 
 
+def test_load_multi_install_package(database, parser, capfd):
+    args = parser.parse_args(['loads', 'mpileaks'])
+    # with pytest.raises(SystemExit):
```

Comment:
Why don't you use `pytest.raises`? Personally I don't see the need to check what is in the error message (which may be changed later).